src/controllers/routers/referees.js

```diff
@@ -88,11 +70,9 @@ const getReferee = async (req, res, next) => {
  */
 const create = async (req, res, next) => {
   const newReferee = sanitize(req.body);
+  console.log(newReferee);
   try {
-    const referee = await Referee.query()
-      .select()
-      .insert(newReferee)
-      .returning('refereeId');
+    const referee = await Referee.query().insert(newReferee).returning('id');
     return res.status(201).json({ referee });
   } catch (err) {
```

Comment:
we should handle NotNullViolationError instances
message should be: <field_name> can't be null

Reply:
create NotNullViolationError instance for create referee